editor/src/components/canvas/remix/remix-utils.spec.browser2.tsx

```diff
@@ -28,6 +31,22 @@ export var storyboard = (
 );
 `
 
+const remixConfigJsFromRemixDocs = `
+/** @type {import('@remix-run/dev').AppConfig} */
+module.exports = {
+  appDirectory: "src",
+  assetsBuildDirectory: "public/build",
+  ignoredRouteFiles: ["**/.*"],
+  publicPath: "/build/",
+  routes(defineRoutes) {
+    return defineRoutes((route) => {
+      route("/somewhere/cool/*", "catchall.tsx");
+    });
+  },
+  serverBuildPath: "build/index.js",
```

Comment:
I'd say that whilst we're only parsing the `appDirectory` we should only include that in this test, otherwise there's a good chance that this sample config will drift from a valid config in a way that we wouldn't catch.

Reply:
great point, fixed in https://github.com/concrete-utopia/utopia/pull/4450/commits/79e549a060bdb3687c2cd2b699e3198a52801ac8